[BUG] Cant connect to BLE bridge.

**Describe the bug**
When I use `meshmonitor-ble-bridge` as `MESHTASTIC_NODE_IP`, i get ENOTFOUND in meshmonitor logs, ble bridge works fine tho. I tried setting IP to `localhost`, but in the result i get ECONREFUSED.

**To Reproduce**
Steps to reproduce the behavior:
1. Pass 'meshmonitor-ble-bridge' to 'MESHTASTIC_NODE_IP' in 'docker-compose.yml'
2. Check meshmonitor logs
3. See error

**Expected behavior**
Connecting to BLE bridge without errors.

**Environment (please complete the following information):**
* Host OS: Linux
* Docker with `ghcr.io/yeraze/meshmonitor:latest`

**Meshmonitor log with error**
[ERROR] ❌ TCP socket error: getaddrinfo ENOTFOUND meshmonitor-ble-bridge
[ERROR] ❌ TCP transport error: getaddrinfo ENOTFOUND meshmonitor-ble-bridge
[ERROR] Failed to connect to Meshtastic node: Error: getaddrinfo ENOTFOUND meshmonitor-ble-bridge
    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'meshmonitor-ble-bridge'
}
[ERROR] Failed to connect to Meshtastic node on startup: Error: getaddrinfo ENOTFOUND meshmonitor-ble-bridge
    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'meshmonitor-ble-bridge'

--- a/docker-compose.ble.yml
+++ b/docker-compose.ble.yml
@@ -16,7 +16,6 @@ services:
     #   dockerfile: Dockerfile
     container_name: meshmonitor-ble-bridge
     privileged: true  # Required for BLE hardware access
-    network_mode: host  # Allows localhost TCP communication with meshmonitor
     restart: unless-stopped
     volumes:
       - /var/run/dbus:/var/run/dbus  # Required for D-Bus/Bluetooth access
@@ -36,8 +35,8 @@ services:
   # Override meshmonitor service to connect to BLE bridge
   meshmonitor:
     environment:
-      # Configure MeshMonitor to connect to the BLE bridge
-      - MESHTASTIC_NODE_IP=localhost
+      # Configure MeshMonitor to connect to the BLE bridge via Docker service name
+      - MESHTASTIC_NODE_IP=meshmonitor-ble-bridge
       - MESHTASTIC_NODE_PORT=4403
     depends_on:
       ble-bridge:

--- a/docs/.vitepress/theme/DockerComposeConfigurator.vue
+++ b/docs/.vitepress/theme/DockerComposeConfigurator.vue
@@ -411,7 +411,6 @@ const dockerComposeYaml = computed(() => {
     lines.push('    image: ghcr.io/yeraze/meshtastic-ble-bridge:latest')
     lines.push('    container_name: meshmonitor-ble-bridge')
     lines.push('    privileged: true')
-    lines.push('    network_mode: host')
     lines.push('    restart: unless-stopped')
     lines.push('    volumes:')
     lines.push('      - /var/run/dbus:/var/run/dbus')

--- a/docs/configuration/ble-bridge.md
+++ b/docs/configuration/ble-bridge.md
@@ -111,7 +111,6 @@ services:
     image: ghcr.io/yeraze/meshtastic-ble-bridge:latest
     container_name: meshmonitor-ble-bridge
     privileged: true  # Required for BLE hardware access
-    network_mode: host  # Allows localhost TCP communication
     restart: unless-stopped
     volumes:
       - /var/run/dbus:/var/run/dbus  # Required for D-Bus/Bluetooth
@@ -129,11 +128,9 @@ The BLE bridge requires `privileged: true` to access Bluetooth hardware. This gr
 
 **Security Note:** Privileged mode should only be used on trusted systems. For production, consider using device-specific capabilities instead.
 
-#### Host Networking
+#### Docker Networking
 
-Using `network_mode: host` allows the BLE bridge to listen on `localhost:4403`, making it accessible to MeshMonitor without complex container networking.
-
-**Alternative:** For isolated networking, see [Advanced Networking](#advanced-networking) below.
+The BLE bridge and MeshMonitor communicate using Docker's internal networking. MeshMonitor connects to the BLE bridge using its container name (`meshmonitor-ble-bridge`) as the hostname.
 
 #### Volume Mounts
 
@@ -148,11 +145,11 @@ When using the overlay, MeshMonitor automatically configures itself:
 
 ```yaml
 environment:
-  - MESHTASTIC_NODE_IP=localhost
+  - MESHTASTIC_NODE_IP=meshmonitor-ble-bridge
   - MESHTASTIC_NODE_PORT=4403
 ```
 
-The BLE bridge acts as a transparent TCP proxy on port 4403.
+The BLE bridge acts as a transparent TCP proxy on port 4403. MeshMonitor connects to it using Docker's internal DNS, which resolves the container name to the correct IP.
 
 ## Pairing Your Device
 

